CONTRIBUTING.md

```diff
@@ -317,7 +318,8 @@ scenarios in this guide.
 
 ### Pull Request Guidelines
 
-See the [pull request guidelines](https://github.com/exercism/docs/blob/master/contributing/pull-request-guidelines.md) in the docs repository.
+See the [pull request guidelines](https://github.com/exercism/legacy-docs/blob/main/contributing/pull-request-guidelines.md)
+in the [legacy-docs](https://github.com/exercism/legacy-docs) repository.
 
 ### Exercise Versioning
```

Comment:
We no longer have versioning for exercise, so this section can be removed (I'd do that in a separate PR though).